fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -551,12 +551,12 @@ ACTOR Future<Void> addBackupMutations(ProxyCommitData* self,
 		wr << (uint8_t)hashlittle(&version, sizeof(version), 0);
 		wr << bigEndian64(commitVersion);
 
-		MutationRef backupMutation;
-		backupMutation.type = MutationRef::SetValue;
 		uint32_t* partBuffer = nullptr;
 
 		for (int part = 0; part * CLIENT_KNOBS->MUTATION_BLOCK_SIZE < val.size(); part++) {
 
+			MutationRef backupMutation;
+			backupMutation.type = MutationRef::SetValue;
```

Comment:
Why do we want to put these inside the forloop? Thanks!

Reply:
Otherwise, the mutation checksum will keep around and we will need to manually call `removeChecksum` for each loop